bin/setup

```diff
@@ -5,6 +5,9 @@ set -e
 gem install bundler --conservative
 bundle check || bundle install
 
+# Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
+mkdir -p .git/safe
```

Comment:
Yup, and while I disagree it should be there as well, it's scarier in an open source project that might be using tb's dotfiles directly. In a closed source project only a few people control it's a more understandable risk to take, but in an open source project with more contributors it scares me too much.